apps/auth-frontend/src/login/Login.tsx

```diff
@@ -7,6 +7,16 @@ import {
 } from '../auth0/web-auth';
 import { extractErrorMessage, WebAuthError } from '../auth0/errors';
 
+const getAppUrl = () => {
+  const redirectUri = new URL(
+    new URLSearchParams(window.location.search).get('redirect_uri') ||
+      'https://hub.asap.science/',
+  );
+  return `${redirectUri.protocol}//${redirectUri.hostname}${
+    redirectUri.port ? `:${redirectUri.port}` : ''
+  }`;
```

Comment:
```suggestion
  return redirectUri.origin;
```
I believe `origin` already returns this.

Reply:
This will also allow removing some test cases that 'test that specific parts of the URL are there' :+1: